custom-graders/v2/DemoPythonGrader/autograder/util.py

```diff
@@ -0,0 +1,24 @@
+# dependencies
+import json, sys, os
+
+# helper function to send print statements to stderr
+def print_stderr(error_msg):
+    print(str(error_msg), file=sys.stderr)
+
+# compile json object for sending score and feedback to Coursera
+def send_feedback(score, msg):
+    post = {'fractionalScore': score, 'feedback': msg}
+    with open("/shared/feedback.json", "w") as outfile:
+        json.dump(post, outfile)
+        print(json.dumps(post))
```

Comment:
@fng-coursera I imagine the print statement is no longer necessary but it is just helpful if surfaced in container logs as well?

Reply:
Yeah that's why I left it there

Reply:
Could we add a quick comment clarifying line 12 and line 13? line 12 is for actual feedback to be surfaced, where as line 13 is just best practice / optional to be sent to container logs. Not sure if that is being overly verbose but wanted to clarify for those who are coming into the demo graders for the first time.

Reply:
Agree. Let us also move this print statement above line#11, and add a comment explaining this will go to container logs and can be used for debugging.

Reply:
Thanks for the feedback. updated accordingly